contract/models/account_analytic_account.py

```diff
@@ -321,7 +326,7 @@ def cron_recurring_create_invoice(self):
             ('date_end', '=', False),
             ('date_end', '>=', today),
         ])
-        return contracts.recurring_create_invoice()
+        return contracts.recurring_create_invoice(limit)
```

Comment:
I didn't want to delay more the 10.0 merge, but now that you are forward-porting (but I don't think it's needed, as as said, we are fixing a problem elsewhere), please put `limit=limit` (always use keyword arguments).

Reply:
That's not needed.

Reply:
we have discussed about this in other occasions. Although you consider this not needed, please do it.